app/uk/gov/hmrc/cipphonenumberverification/services/OtpService.scala

```diff
@@ -17,10 +17,11 @@
 package uk.gov.hmrc.cipphonenumberverification.services
 
 import java.security.SecureRandom
+import scala.collection.mutable
 
 object OtpService {
-  def otpGenerator = {
-    val sb = new StringBuilder()
+  def otpGenerator: String = {
+    val sb = new mutable.StringBuilder()
```

Comment:
why should this be mutable when its already mutable

Reply:
was just intellij suggestion